ddtelemetry/src/worker/mod.rs

```diff
@@ -675,6 +675,8 @@ impl TelemetryWorkerHandle {
                 message,
                 level,
                 stack_trace,
+                tags: String::new(),
+                is_sensitive: false,
```

Comment:
should this default to false? Where would we have `is_sensitive: true`?

Reply:
I set it to false because the existing behavior is that telemetry logs are not assumed to contain sensitive data.